CMakeSettings.json

```diff
@@ -137,11 +132,6 @@
           "name": "BUILD_PERFORMANCE_TESTS",
           "value": "True",
           "type": "BOOL"
-        },
-        {
-          "name": "CMAKE_BUILD_TYPE",
-          "value": "MinSizeRel",
-          "type": "STRING"
```

Comment:
Why did you remove this configuration from the Perf profiles?  This is the Release config for Perf.